examples/chip-tool-darwin/commands/clusters/ModelCommandBridge.h

```diff
@@ -31,16 +31,16 @@ class ModelCommand : public CHIPCommandBridge
     void AddArguments()
     {
         AddArgument("node-id", 0, UINT64_MAX, &mNodeId);
-        AddArgument("endpoint-id", 0, UINT16_MAX, &mEndPointId);
+        AddArgument("endpoint-id", 0, UINT16_MAX, &mEndPointIds);
```

Comment:
Is `AddArguments()` here coming from `examples/chip-tool/commands/common/Command.h`?  If so, shall we add the optional `desc` description string to the arguments we are modifying in this review?